thirdeye/thirdeye-frontend/app/pods/manage/alerts/index/template.hbs

```diff
@@ -127,9 +127,9 @@
                 {{/link-to}}
               </div>
             </div>
-            <div class="te-search-results__edit-button">
+            {{!-- <div class="te-search-results__edit-button">
               {{#link-to "manage.alerts.edit" alert.id}}<span class="glyphicon glyphicon-pencil"></span>{{/link-to}}
-            </div>
+            </div> --}}
```

Comment:
Do you still need this, or is it commented out for something? If it's the latter, let's add a `TODO` for the task or comments on why it's uncommented.

Reply:
Adding a note for this - I want to keep it there for now.